paper-tooltip.sublime-snippet

```diff
@@ -0,0 +1,16 @@
+<snippet>
+	<content><![CDATA[
+<paper-tooltip
+  for="$1"
+  position="${2:bottom}"
+  offset="${3:14}"
+  >
```

Comment:
can you move this to the same line as "offset". That's how the other snippets are formatted